suricata/update/main.py

```diff
@@ -50,18 +50,20 @@
     sys.path.insert(
         0, os.path.abspath(os.path.join(__file__, "..", "..", "..")))
 
-import suricata.update.rule
-import suricata.update.engine
-import suricata.update.net
-import suricata.update.loghandler
-from suricata.update import config
-from suricata.update import configs
-from suricata.update import extract
-from suricata.update import util
-from suricata.update import sources
-from suricata.update import commands
-from suricata.update import exceptions
-from suricata.update import notes
+from suricata.update import (
+    rule as rule_mod,
+    engine,
+    net,
+    loghandler,
+    config,
+    configs,
+    extract,
+    util,
+    sources,
+    commands,
+    exceptions,
+    notes,
+)
```

Comment:
Can we also have these in lexicographical order? Rest looks good.

Reply:
@shivan1b Thanks for the feedback. I will make these sorted.